_rules/iframe-accessible-name-cae760.md

```diff
@@ -150,12 +151,22 @@ Does not apply to non `iframe` element.
 
 #### Inapplicable Example 2
 
-`iframe` is not [included in the accessibility tree][].
+This `iframe` is not part of [sequential focus navigation][], nor [included in the accessibility tree][] because of `display: none;`.
```

Comment:
As far as I understand, browsers that "force push" iframes in sequential navigation won't do it here because of the `display:none`, which is fine.
The description, however, looks a bit like the "because of `display:none` is related only to the second part (not in the accessibility tree) rather than to both.

But I don't really have any idea to improve it…

Reply:
OK, leaving it as is, since it got 2 approvals.